go.mod

```diff
@@ -7,8 +7,9 @@ require (
 	github.com/go-openapi/spec v0.19.4
 	github.com/go-playground/validator/v10 v10.3.0
 	github.com/iancoleman/strcase v0.0.0-20190422225806-e506e3ef7365
+	github.com/integr8ly/grafana-operator/v3 v3.4.0
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/operator-framework/operator-sdk v0.15.2
+	github.com/operator-framework/operator-sdk v0.16.0
```

Comment:
The Operator SDK should be updated in Makefile and than `make codegen` need to be run

Reply:
Aha, didn't know this. Let me look into it.